src/components/Category.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled'
+import { categories } from '@/data/categories'
+
+export default function Category() {
+  return (
+    <>
+      <Block/>
+      <Section>
+        <Title>선물 테마</Title>
+        <Grid>
+          {categories.map((item) => (
+            <CategoryItem key={item.themeId}>
+              <Image src={item.image} alt={item.name} />
+              <Label>{item.name}</Label>
+            </CategoryItem>
```

Comment:
이걸 다 묶어서 CategorItem 으로 표현해도 좋을것 같아요!
```suggestion
            <CategoryItem key={item.themeId} {...item} />
```
요로코롬...